src/test/java/uk/gov/hmcts/reform/sendletter/controllers/GetLetterStatusControllerTest.java

```diff
@@ -39,49 +41,55 @@
     @MockBean private AuthTokenValidator tokenValidator;
     @MockBean private AuthChecker authChecker;
 
-    private UUID letterId;
+    private LetterStatus letterStatus;
 
     @Before
     public void setUp() {
-        letterId = UUID.randomUUID();
+        LocalDateTime now = LocalDateTime.now();
+        letterStatus = new LetterStatus(UUID.randomUUID(), "some-message-id", now, now, now);
     }
 
     @Test
     public void should_return_letter_status_when_it_is_found_in_database() throws Exception {
+
         given(tokenValidator.getServiceName("auth-header-value")).willReturn("service-name");
-        given(service.getStatus(letterId, "service-name")).willReturn(new LetterStatus(letterId));
+        given(service.getStatus(letterStatus.id, "service-name")).willReturn(letterStatus);
 
-        getLetter(letterId)
+        getLetter(letterStatus.id)
             .andExpect(status().isOk())
             .andExpect(content().json(
                 "{"
-                    + "\"id\":\"" + letterId.toString() + "\""
+                    + "\"id\":\"" + letterStatus.id.toString() + "\","
+                    + "\"message_id\":\"" + letterStatus.messageId + "\","
+                    + "\"created_at\":\"" + letterStatus.createdAt.toString() + "\","
+                    + "\"sent_to_print_at\":\"" + letterStatus.sentToPrintAt.toString() + "\","
+                    + "\"printed_at\":\"" + letterStatus.printedAt.toString() + "\""
```

Comment:
is this ISO_8601?
Would be good to have actual strings here imho.

Reply:
i see your point. i only received error because of extra lib introduced and config not present. good i did actually. but actual strings are better proof :+1: